crates/service/src/index/segments/sealed.rs

```diff
@@ -58,6 +58,22 @@ impl<S: G> SealedSegment<S> {
     pub fn len(&self) -> u32 {
         self.indexing.len()
     }
+    pub fn size(&self) -> SegmentSizeInfo {
+        let mut info = SegmentSizeInfo {
+            id: self.uuid,
+            typ: "sealed".to_string(),
+            length: self.len() as usize,
+            size: 0,
+        };
+        let path = self._tracker.path.join("indexing");
+        match dir_size(&path) {
+            Ok(size) => info.size = size as u64,
+            Err(e) => {
+                log::error!("Failed to get size of {:?}: {}", path, e);
```

Comment:
Panic might be better?

Reply:
Collecting the OS file info may fail due to numerous reasons. Will the service panic be caught by pgrx and reflected on the postgres part?

Reply:
For the background worker process, if spawned threads panic, it just prints the log and exit the current session (so the transaction aborts). So it's OK to panic.

Reply:
Anyway it should not return an incorrect size.

Reply:
Already changed to panic.